components/TopNav.vue

```diff
@@ -1,18 +1,18 @@
 <template>
-    <div class="p-4 border-b flex justify-between items-center">
+    <div class="p-4 border-b flex flex-row justify-between items-center bg-indigo-800">
+        <HamburgerComponent />
         <div class="font-semibold text-xl text-gray-800">
             <NuxtLink class="align-middle" to="/">
                 <img class="inline align-middle pr-2"
                     src="../assets/images/logo-32-px-color.svg" />Find a Doc
             </NuxtLink>
         </div>
-        <div class="flex gap-4 mx-6 absolute right-20">
-            <NuxtLink to="/aboutPage" class="px-2 mx-auto">
-                About Us
+        <nav class="flex gap-4 mx-6 absolute right-20">
+            <NuxtLink to="/about" class="px-2 mx-auto">
+                About
             </NuxtLink>
-
```

Comment:
I moved the Hamburger to the Upper Left Side, which a common location

Reply:
Will it conflict with the logo? Or is it in place of